packages/react/src/components/RadioGroup/index.tsx

```diff
@@ -17,137 +17,145 @@ export interface RadioGroupProps {
+    const inputs = useRef<HTMLInputElement[]>([]);
+    const handleChange = (value: any) => setCurrentValue(value);
+    const onRadioFocus = (index: number) => setFocusIndex(index);
+    const onRadioBlur = () => setFocusIndex(null);
+    const onRadioClick = (index: number) => {
+      const radio = inputs.current?.[index];
+
+      if (!radio) {
+        return;
+      }
+
+      radio.click();
+      radio.focus();
+    };
+
+    useEffect(() => {
+      if (typeof value === 'undefined') {
+        return;
+      }
+
+      setCurrentValue(value);
+    }, [value]);
+
+    const radioButtons = radios.map((radio, index) => {
+      const {
+        label,
+        disabled,
+        value: radioValue,
+        labelDescription,
+        id,
+        className,
+        ...other
+      } = radio;
+      const isChecked = currentValue === radioValue;
+      const isFocused = focusIndex === index;
+
+      return (
+        <div className="Radio__wrap" key={id}>
+          <div className={classNames('Radio is--flex-row', className)}>
+            <input
+              type="radio"
+              name={name}
+              value={radioValue}
+              id={id}
+              ref={input => {
+                if (!input) {
+                  return;
+                }
+
+                inputs.current.push(input);
+              }}
+              onFocus={() => onRadioFocus(index)}
+              onBlur={() => onRadioBlur()}
+              onChange={() => {
+                handleChange(radioValue);
+                onChange(radio, inputs.current?.[index]);
+              }}
+              disabled={disabled}
+              checked={isChecked}
+              aria-describedby={labelDescription ? `${id}Desc` : undefined}
+              {...other}
+            />
+            <label
+              htmlFor={id}
+              className={classNames('Radio__label', {
+                'Field__label--disabled': disabled
+              })}
+            >
+              {label}
+            </label>
+            <Icon
+              className={classNames('Radio__overlay', {
+                'Radio__overlay--focused': isFocused,
+                'Radio__overlay--disabled': disabled
+              })}
+              type={isChecked ? 'radio-checked' : 'radio-unchecked'}
+              aria-hidden="true"
+              onClick={() => onRadioClick(index)}
+            />
+          </div>
+          {labelDescription && (
+            <span
+              id={`${id}Desc`}
+              className={classNames('Field__labelDescription', {
+                'Field__labelDescription--disabled': disabled
+              })}
+            >
+              {labelDescription}
+            </span>
+          )}
+        </div>
+      );
+    });
+
+    // Hack to prevent ESLint from erroring about this variable not
+    // being used. We want to pull it from `props` to ensure it's
+    // not passed through to the radiogroup element.
+    void defaultValue;
```

Comment:
I think what we can do instead is set `// eslint-disable-next-line no-unused-vars` for `defaultValue`. I'm not sure why we did this.

Reply:
Isn't it being used [on this line](https://github.com/dequelabs/cauldron/blob/423688b873a3da251bc7e2ea4ab874acf654f63a/packages/react/src/components/RadioGroup/index.tsx#L38-L40)?

I tried taking this line out completely, and eslint didn't complain, is that okay?

Reply:
Ah yeah, that's fine. I think that was added after we put in this hack so we should be good to remove.